src/data/markdown/docs/02 javascript api/06 k6-execution.md

```diff
@@ -74,6 +74,13 @@ Meta information and execution details about the current vu and iteration.
 | iterationInScenario | integer | The identifier of the iteration in the current scenario. |
 | idInInstance        | integer | The identifier of the VU across the instance. |
 | idInTest            | integer | The globally unique (across the whole test run) identifier of the VU. |
+| tags                | object  | It gives control over VU's Tags. The Tag will be included in every metric emitted by the VU and the Tag's state is maintained across different iterations until the VU exists. |
```

Comment:
I've noticed this the previous time too but didn't get immediately what the problem is: all other items in the table start with a noun :sweat_smile: So to keep more in line, maybe like this:
```suggestion
| tags                | object  | The map that gives control over VU's Tags. The Tags will be included in every metric emitted by the VU and the Tags' state is maintained across different iterations until the VU exists. |
```